tests/test_models.py

```
@@ -140,3 +140,98 @@ def test_agent_template_request():
+    """Test description whitespace trimming."""
+    from rouge.core.models import Patch
+
+    patch = Patch(id=1, issue_id=10, description="  Fix typo  ")
+    assert patch.description == "Fix typo"
+
+
+def test_patch_empty_description_validation():
+    """Test that empty description raises validation error."""
+    from rouge.core.models import Patch
+
+    with pytest.raises(ValueError):
```

Ruff (0.14.11)

`pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)